Fakebook.Posts/Fakebook.Posts.UnitTests/DataAccess.Model.Test/Comment.cs

```diff
@@ -0,0 +1,119 @@
+﻿using Fakebook.Posts.DataAccess.Models;
+using System;
+using Xunit;
+
+namespace Fakebook.Posts.UnitTests.DataAccess_Model_Test
+{
+    public class CommentTest : IDisposable
+    {
+        Comment testComment;
+        public CommentTest()
+        {
+            testComment = new Comment();
+
+        }
+        /// <summary>
+        /// Test the property Id. Ensures that the model will get and set the id.
+        /// </summary>
+        [Fact]
+        public void CommentTest1()
+        {
+
+            //Act
+            try {
+                testComment.Id = 1;
+            } catch (NotImplementedException) { }
+
+
+            //Assert
+            Assert.Equal(1, testComment.Id);
+        }
+        /// <summary>
+        /// Test the property content. Ensures that the model will get and set the content.
+        /// </summary>
+        [Fact]
+        public void CommentTest2()
+        {
+            //Arrange
+
+            //Act
+            try {
```

Comment:
what is this? this isn't right. adding new tests is not for tdd-pass PRs. and a test that checks for notimplementedexception is not a good test since that isn't desirable behavior. i don't see what's complicated about this. remove the try catches and change the label to tdd-fail

Reply:
Reverted addition of try catch blocks and labeled as tdd-fail